docs/source/setup/ecosystem.rst

```diff
@@ -0,0 +1,26 @@
+Isaac Lab Ecosystem
+===================
+
+Isaac Lab is built on top of Isaac Sim to provide a unified and flexible framework
+for robot learning that exploits latest simulation technologies. It is designed to be modular and extensible,
+and aims to simplify common workflows in robotics research (such as RL, learning from demonstrations, and
+motion planning). While it includes some pre-built environments, sensors, and tasks, its main goal is to
+provide an open-sourced, unified, and easy-to-use interface for developing and testing custom environments
+and robot learning algorithms.
+
+Working with Isaac Lab requires the installation of Isaac Sim, which is packaged with core robotics tools
+that Isaac Lab depends on, including URDF and MJCF importers, simulation managers, and ROS features. Isaac
+Sim also builds on top of the Nvidia Omniverse platform, leveraging advanced physics simulation from PhysX,
+photorealistic rendering technologies, and Universal Scene Description (USD) for scene creation.
+
+Isaac Lab not only inherits the capabilities of Isaac Sim, but also adds a number
+of new features that pertain to robot learning research. For example, including actuator dynamics in the
+simulation, procedural terrain generation, and support to collect data from human demonstrations.
+
+.. figure:: ../_static/setup/ecosystem.jpg
```

Comment:
We need an image for the dark theme as well.

Reply:
Also would be nice to arrange the blocks in the figure to make it coherent. Right now it appears like a jumble of different things (RL on one corner and IL on the other). IMO we should have them one under the other. Basically it should be easier to understand if things that are "similar" are next to each other